### ISSUE TITLE
Missing border radius and borders when switching from Master or Dwindle with no_gaps_when_only set

### ISSUE THREAD

**Issue by BinaryInk** [dawsers/hyprscroller#63]
# Issue

Border radius and border window decorations are not applied when switching to Scroller from the built-in Master and Dwindle layouts when `no_gaps_when_only` is non-zero; which decorations are not applied depends on the value set. 1 only affects border radius; 2 affects both border radius and border.

# Steps to Reproduce

1. Set `no_gaps_when_only` to 1 or 2 for Master or Dwindle layout.
2. Open a single window on a workspace while in Master or Dwindle layout.
3. Using hyprctl or a bind, switch to the Scroller layout.
4. If you switch back to Master/Dwindle and open a second window or make the single window floating, then switch back to 

- If `no_gaps_when_only` is `1` or `2`, there will be no border radius applied.
- If `no_gaps_when_only` is `2`, there will be no border applied.
- Newly spawned windows are not affected and keep their border (unless it is the only window and you go back to Master/Dwindle and back to Scroller again)

**Comment by dawsers** [comment]
When Master or Dwindle see `no_gaps_when_only`, they apply the option to the window, and "steal" the border 

``` cpp
PWINDOW->m_sWindowData.noBorder   = CWindowOverridableVar(*PNOGAPSWHENONLY != 2, PRIORITY_LAYOUT);
```

Windows are global entities, so if then you pass the window to hyprscroller, it has no way to know why the window doesn't have a border. The reason may be not just that Master/Dwndle "stole" it, but it could be a workspace rule or a window rule.

So I would argue it is a bug in Master/Dwindle. It is not a general option, but particular to those layouts, so I have no reason to parse it. And they should restore the default value when the layout is disabled/unloaded.

Let me know what you think.

**Comment by dawsers** [comment]
Fixed in d59efa56226c0c28ac6e49445cfd2153c479f867
